23-freight-car-door-motion-and-location/config.json

```diff
@@ -0,0 +1,14 @@
+# uncomment set your own product UID and serial number
+{ "req": "hub.set", "sn": "my-serial", "product": "com.your-company:your-product-name" }
+{ "req": "hub.set", "mode": "periodic", "inbound": 60, "outbound": 60 }
+{ "req": "card.voltage", "mode": "lipo" }
+
+# track location daily
+{ "req": "card.location.mode", "mode": "periodic", "vseconds": "usb:30;high:86400;normal:86400;low:178200;dead:0" }
+{ "req": "card.location.track", "start": true }
+
+# Monitor door opening via the Hall sensor/reed switch.
```

Comment:
I assume the magnetic door switch is the reed switch, but just noting that you haven't used that term until this point.

Reply:
Good catch. I should be consistent.